src/core/style/_mixins.scss

```diff
@@ -7,12 +7,12 @@
   position: relative;
 
   // Use a transform to create a new stacking context.
-  transform: translate3D(0, 0, 0);
+  transform: translate3d(0, 0, 0);
 }
 
 /**
  * This mixin hides an element visually.
- * That means it's still accessible for screen-readers but not visible in view.
+ * That means it"s still accessible for screen-readers but not visible in view.
```

Comment:
Ah, that was the one I mentioned - forgot to re-push after fixing it =x

On Tue, May 31, 2016 at 5:57 PM, Ivey Padgett <notifications@github.com>
wrote:

> In src/core/style/_mixins.scss
> <https://github.com/angular/material2/pull/590#discussion_r65288028>:
>
> >  }
> >
> >  /**
> >   * This mixin hides an element visually.
> > - * That means it's still accessible for screen-readers but not visible in view.
> > + * That means it"s still accessible for screen-readers but not visible in view.
>
> Missed a double quote 😄
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/material2/pull/590/files/8fb026e39355a422b9e34fac2987bf85a0af9a68#r65288028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAi34wSiIQqHKa2z5pqu5R3bOuSki3rLks5qHNjogaJpZM4Iq1V->
> .
>